### PR TITLE
Add 2.6.2 release blog

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

### Modifications

Add 2.6.2 release blog

### CODE REVIEW

File: site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+For more information about implementation, see [PR-7335](https://github.com/apache/pulsar/pull/7335).
+
+### Converted the namespace bundle unload into async operation
+
+Reworked the namespace bundle unload REST handler to be asynchronous. This avoid getting stuck in the Jetty threads calling into themselves when we try to unload the entire namespace.
+
+For more information about implementation, see [PR-7364](https://github.com/apache/pulsar/pull/7364).
+
+### Support initial namespace of the cluster without startup the broker
```
Comment (eolivelli) [inline site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md:34]: Typo: startup -> starting

**Review by eolivelli — APPROVED** [review]

File: site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md (hunk 1 above)
Comment (wolfstudy) [reply site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md]: Cool, thanks @eolivelli

**Review by jiazhai — APPROVED** [review]

**Comment by codelipenghui** [comment]
@Jennifer88huang @Huanli-Meng Could you please help review the release blog for 2.6.2?

**Comment by Jennifer88huang** [comment]
@codelipenghui sure. Will give feedback soon.

File: site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+
+Here are some highlights and major features added in Pulsar 2.6.2.
+
+<!--truncate-->
+
+## Broker
+
+### Catch throwable when start pulsar
+
+Before 2.6.1, pulsar only catch exception when `BrokerStarter.start()` failed, some error such as `NoSuchMethodError` or `NoClassDefFoundError` can not be caught, and pulsar will in abnormal status but no error log will found in the log file.
```
Comment (Jennifer88huang) [inline site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md:16]: 2.6.1 or 2.6.2?

File: site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+
+Before 2.6.1, pulsar only catch exception when `BrokerStarter.start()` failed, some error such as `NoSuchMethodError` or `NoClassDefFoundError` can not be caught, and pulsar will in abnormal status but no error log will found in the log file. 
+
+In 2.6.1, we modify exception to throwable to avoid this issue.
+
+For more information about implementation, see [PR-7221](https://github.com/apache/pulsar/pull/7221).
+
+### Handle SubscriptionBusyException in resetCursor API
+
+In method `PersistentSubscription.resetCursor`, `SubscriptionFencedException` is thrown in several place, but not handled in `PersistentTopicBase`. It caused the error message not very clear. This fix tries to export `SubscriptionBusyException` in PersistentTopicBase for resetCursor, to make the error in reset API more clear.
```
Comment (Jennifer88huang) [inline site2/website/blog/2020-11-09-Apache-Pulsar-2-6-2.md:24]: In Rest API or in reset API?

**Comment by wolfstudy** [comment]
@Jennifer88huang PTAL again thanks.

**Comment by Jennifer88huang** [comment]
@wolfstudy I've edited some minor issues directly. Please double confirm. Thank you.

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**